### ISSUE TITLE
Cannot resolve to parent directory via tsconfig path in subdirectories

### ISSUE THREAD

**Issue by Tanimodori** [johnsoncodehk/volar#1596]
This problem occurred in a fresh clone of [vite-electron-builder](https://github.com/cawa-93/vite-electron-builder). The path alias to parent directory cannot be resolved while it can run with no problem. 

Steps to reproduce

1. `git clone https://github.com/cawa-93/vite-electron-builder.git`
2. `npm install`
3. Open `packages\renderer\src\components\ElectronVersions.vue`
4. The `import {versions} from '#preload'` shows error when resolving

![Code_fFKOnLexZO](https://user-images.githubusercontent.com/29478770/179658955-f82a327f-3071-4403-8413-a6ba0a13fe58.png)

However, run `npm run watch` and this project is able to run.

The `tsconfig.json` file path seems to be correct.

![Code_JQNSh1LFsp](https://user-images.githubusercontent.com/29478770/179659675-7d0e2069-af3b-4ede-94f8-7d951ff93bb8.png)

Relating `tsconfig.json` (`packages\renderer\tsconfig.json`) sections:

```json
    "paths": {
      "#preload": ["../preload/src/index"],
      "/@/*": ["./src/*"]
    },
```

Even change the import path to `../../../preload/src/index` works:

![Code_CPrLFDeBP9](https://user-images.githubusercontent.com/29478770/179659352-cd203710-6895-4ea2-9aed-54d2a422657a.png)

Environment: volar v0.38.8, VSCode 1.69.2 on Windows 10, TS 4.7.3 in takeover mode (turn off does not help).

**Comment by Tanimodori** [comment]
MIGHT be related issue: https://github.com/cawa-93/vite-electron-builder/issues/110

**Comment by Tanimodori** [comment]
Fixed in volar v0.38.9
